src/components/Button/Button.spec.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+
+import { renderWithTheme, theme } from '../../../test/utils'
+import { blockSizes } from '../common/system'
+
+import Button from './Button'
+
+const defaultProps = {
+  children: 'click me'
+}
+
+describe('<Button />', () => {
+  it('should render button', () => {
+    const { getByRole } = render(<Button {...defaultProps} />)
+    const button = getByRole('button')
+
+    expect(button).toBeInTheDocument()
+    expect(button.tagName).toBe('BUTTON')
+  })
+
+  it('should handle different types', () => {
+    const { getByRole } = render(<Button {...defaultProps} type='submit' />)
+    const button = getByRole('button')
+
+    expect(button).toHaveAttribute('type', 'submit')
+  })
+
+  it('should handle click properly', () => {
```

Comment:
We should also test if the button with `disabled` prop reacts to onClick (when button is disabled it doesn't trigger specified `onClick` prop when clicked).

Reply:
well-observed, thanks!

fixed :)